### PR TITLE
[13.0][MIG] l10n_fi_payment_reference: Migration and a rework to 13.0

### PR DESCRIPTION
**Pull request by Menestrels**

### CODE REVIEW

File: l10n_fi_payment_reference/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Payment Reference Type can be selected from Accounting Settings.
```
Comment (mlaitinen) [inline l10n_fi_payment_reference/readme/CONFIGURE.rst]: Please update to match the current implementation

File: l10n_fi_payment_reference/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+The module allows users to generate a valid invoice payment reference for sales invoices.
```
Comment (mlaitinen) [inline l10n_fi_payment_reference/readme/DESCRIPTION.rst]: Please update to match the current implementation. The payment reference is already there, so this description doesn't really match the contents of this module.

**Review by mlaitinen — CHANGES_REQUESTED** [review]
just a few changes to the doc and we can merge this

File: l10n_fi_payment_reference/readme/DESCRIPTION.rst (hunk 2 above)
Comment (Menestrels) [reply l10n_fi_payment_reference/readme/DESCRIPTION.rst]: I updated description

File: l10n_fi_payment_reference/readme/CONFIGURE.rst (hunk 1 above)
Comment (Menestrels) [reply l10n_fi_payment_reference/readme/CONFIGURE.rst]: Updated

**Pull request merged**